fingerprint2.js

```diff
@@ -217,9 +217,19 @@
       return keys;
     },
     webglKey: function(keys) {
-      if(!this.options.excludeWebGL && this.isCanvasSupported()) {
+      var isWebGlExcluded = !!this.options.excludeWebGL,
+          isWebGlSupported = this.isWebGlSupported();
+
+      if (!isWebGlExcluded && isWebGlSupported) {
```

Comment:
I think it's a mistake. You test for support of WebGL and then log that it's not supported.
Could you also please not introduce the isWebGlExcluded variable and operate solely on original options.excludeWebGL? I think it can be done w/out adding the variable.